wkcuber/README.md

```diff
@@ -159,25 +53,6 @@ poetry run pylint -j4 wkcuber
 poetry run pytest tests
 ```
 
-Please, run the extended test suite:
-```bash
-tests/scripts/all_tests.sh
```

Comment:
What about these tests? Were they removed? The `all_tests.sh` file is not part of the PR and remains unchanged.